src/Form/AutologoutSettingsForm.php

```diff
@@ -248,21 +248,20 @@ public function validateForm(array &$form, FormStateInterface $form_state) {
 
     if($values['role_logout']) {
       // Validate timeouts for each role.
-      foreach (user_roles(TRUE) as $key => $role) {
-        if (empty($new_stack[$key])) {
+      foreach (user_roles(TRUE) as $role => $role_object) {
+        if (empty($new_stack[$role]) || $new_stack[$role]['enabled'] == 0 ) {
```

Comment:
The data that we want here is the key of the array, hence named it as $role, which is used in Line no 260. $role_object is not really used any where.